packages/circleci/src/circleci-config.ts

```diff
@@ -137,7 +156,7 @@ export default abstract class CircleCiConfigHook {
       config.orbs['tool-kit'] = `financial-times/dotcom-tool-kit@${currentVersion}`
     }
 
-    if (!(config.workflows?.['tool-kit'] as Workflow).jobs) {
+    if (!(config.workflows?.['tool-kit'] as Workflow).jobs || !(config.workflows?.['nightly'] as Workflow).jobs) {
```

Comment:
**issue:** shouldn't this be checking that both of the workflows are valid, not either?
```suggestion
    if (!(config.workflows?.['tool-kit'] as Workflow).jobs && !(config.workflows?.['nightly'] as Workflow).jobs) {
```

Reply:
nice catch!! thank you!

Reply:
nice catch!! thank you!